worker/src/lib/HeliconeHeaders.ts

```diff
@@ -104,7 +104,7 @@ export class HeliconeHeaders implements IHeliconeHeaders {
       openaiBaseUrl: this.headers.get("Helicone-OpenAI-Api-Base") ?? null,
       retryHeaders: this.getRetryHeaders(),
       promptFormat: this.headers.get("Helicone-Prompt-Format") ?? null,
-      requestId: this.headers.get("Helicone-Request-Id") ?? null,
+      requestId: this.headers.get("Helicone-Request-Id") ?? crypto.randomUUID(),
```

Comment:
nice catch. I am pretty sure we do this somewhere else - nice to do it earlier thought